site/blog/2022/nodejs-deploy-codepipeline.mdx

```diff
@@ -0,0 +1,221 @@
+---
+slug: deploy-nodejs-app-with-codepipeline
+title: Deploy a NodeJS App on AWS using CodePipeline
+date: 2022-12-23
+authors: [yrobla]
+---
+
+# Deploy a NodeJS app to AWS with one click using CodePipeline
+
+Codepipeline is a service provided by AWS to provide a fully automated Continuous delivery service. It allows to fully manage all the steps in the code delivery lifecycle, from checking the source code and building it, testing and deploying to production. The whole process is managed by CodePipeline using specific configuration files without manual intervention, offering a safe and reliable way to deploy to the cloud.
```

Comment:
Added a link to better define Continuous Deployment, title-cased that word, minor grammar changes, and made the capitalization of CodePipeline consistent with how AWS does it.

```suggestion
CodePipeline is a service provided by AWS to provide a fully automated [Continuous Delivery](https://en.wikipedia.org/wiki/Continuous_delivery) service. It allows you to fully manage all the steps in the code delivery lifecycle, from checking out the source code, building it, testing it, and deploying to production. The whole process is managed by CodePipeline using specific configuration files without manual intervention, offering a safe and reliable way to deploy to the cloud.
```